cypress/support/fragments/users/userDefaultObjects/loanDetails.js

```diff
@@ -200,4 +200,18 @@ export default {
       },
     );
   },
+  openFeeFine(amount) {
+    cy.do(Button({ className: including('feefineButton-') }, including(amount)).click());
+  },
+  checkSourceName(row, sourceName) {
```

Comment:
This method is pretty match similar to existing one. Can these two methods be rewritten to move common logic to single source?